iphone/Classes/TiUIWindowProxy.m

```diff
@@ -704,6 +705,12 @@ - (void)setBackButtonTitleImage:(id)proxy
   }
 }
 
+- (void)setHideBackButton:(id)arg
```

Comment:
You use `hideBackButton` here, but `hidesBackButton` above. Also, `navigationItem` can be `nil` on window instances without navigation controllers. Do we even need this method here?

Reply:
Its not required. Added for testing but forget to remove. Now removed. Good Catch!